### PR TITLE
fix(patch): Remove useless dependency.

### PR DESCRIPTION
**Pull request by ser-0xff**
## Description

Remove useless dependency.

## How Has This Been Tested?

Unit tests.

## Minimal checklist:

- [X] I have performed a self-review of my own code 
- [ ] I have added `DocC` code-level documentation for any public interfaces exported by the package
- [ ] I have added unit and/or integration tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: Sources/DistributedSystem/DistributedSystem.swift [hunk 1]
```diff
@@ -30,6 +29,14 @@ extension Channel {
     }
 }
 
+public final class Box<T> {
```
Comment (dimlio) [inline Sources/DistributedSystem/DistributedSystem.swift:32]: Needs to be public?

File: Sources/DistributedSystem/TransferableConformance/Dictionary+Transferable.swift [hunk 2]
```diff
@@ -6,10 +6,12 @@
 //
 // http://www.apache.org/licenses/LICENSE-2.0
 
-import Helpers
-
 fileprivate typealias SizeType = UInt32
 
+func nearestPowerOf2<T: FixedWidthInteger>(_ num: T) -> T {
```
Comment (dimlio) [inline Sources/DistributedSystem/TransferableConformance/Dictionary+Transferable.swift:11]:
```suggestion
private func nearestPowerOf2<T: FixedWidthInteger>(_ num: T) -> T {
```
if used only here?

**Review by dimlio — APPROVED** [review]

File: Sources/DistributedSystem/DistributedSystem.swift (hunk 1 above)
Comment (ser-0xff) [reply Sources/DistributedSystem/DistributedSystem.swift:32]: Yes, it there is a public member variable of that type...

File: Sources/DistributedSystem/TransferableConformance/Dictionary+Transferable.swift (hunk 2 above)
Comment (ser-0xff) [reply Sources/DistributedSystem/TransferableConformance/Dictionary+Transferable.swift:11]: No, it is used in some other places.

**Pull request merged**